frontend/src/Components/Form/EnhancedSelectInput.js

```diff
@@ -6,16 +6,16 @@ import classNames from 'classnames';
 import getUniqueElememtId from 'Utilities/getUniqueElementId';
 import isMobileUtil from 'Utilities/isMobile';
 import * as keyCodes from 'Utilities/Constants/keyCodes';
-import { icons, scrollDirections } from 'Helpers/Props';
+import { icons, sizes, scrollDirections } from 'Helpers/Props';
 import Icon from 'Components/Icon';
 import Portal from 'Components/Portal';
 import Link from 'Components/Link/Link';
 import Measure from 'Components/Measure';
 import Modal from 'Components/Modal/Modal';
 import ModalBody from 'Components/Modal/ModalBody';
 import Scroller from 'Components/Scroller/Scroller';
-import EnhancedSelectInputSelectedValue from './EnhancedSelectInputSelectedValue';
-import EnhancedSelectInputOption from './EnhancedSelectInputOption';
```

Comment:
Root folder select component is the only other component that uses these two, would it make sense to switch it to the new hinted value/option components and purge these? Or just build the hinted functionality into the existing components instead of making new named components?

Reply:
The Hinted one creates a specific layout that can't be overridden since it has a div with two items in it with a specific flex and style. If a future component wants to do a different layout. For example an icon. Then it wouldn't be able to properly layout the dropdown component, particularly on mobile.
That's why the base components are separate.